cinder/policies/services.py

```diff
@@ -76,6 +77,16 @@
                 'path': '/os-services/failover_host'
             }
         ]),
+    policy.DocumentedRuleDefault(
+        name=RECOUNT_VOLUMES_POLICY,
+        check_str=base.RULE_ADMIN_API,
```

Comment:
I'm not sure if the `rule:admin_api` is enough, as I couldn't find out how `is_admin_project` is set - it seems to default to `True` in some cases. Therefore, we better add this policy to the helm-charts and set it to `context_is_admin`

Reply:
only admins can call this api.